pkg/runtime/wfengine/workflow.go

```diff
@@ -266,6 +268,7 @@ func (wf *workflowActor) purgeWorkflowState(ctx context.Context, actorID string)
 	return nil
 }
 
+// [Question] What could be the use-case of calling this function ?
 func (wf *workflowActor) addWorkflowEvent(ctx context.Context, actorID string, historyEventBytes []byte) error {
```

Comment:
`addWorkflowEvent` is called to trigger further processing in workflow. The event could be like TerminateEvent, SuspendEvent, RaiseEvent etc. See this: https://github.com/dapr/dapr/tree/master/pkg/runtime/wfengine#resiliency.

Reply:
This makes total sense now.